features/cli.feature

```diff
@@ -3,6 +3,21 @@ Feature: Command line interface
   As a person who wants to run features
   I want to run Cucumber on the command line
 
+  Scenario: run with no features directory
+    When I run cucumber.js with `features`
+    Then the error output contains the text:
+    """
+    No such file or directory - features. You can use `cucumber --init` to get started.
+    """
+
+  Scenario: run a nonexistant feature file
+    Given a directory named "features" 
+    When I run cucumber.js with `features/a.feature`
+    Then the error output contains the text:
+    """
+    No such file or directory - a.feature
```

Comment:
I'd like this to print the path relative to the current working directory. If you run 
```
cucumber-js features/some/nested/a.feature features/some/other/nested/a.feature
```
If one feature doesn't exist and we print
```
No such file or directory - a.feature
```
You don't know which one. A bit of an extreme example, but I hope it illustrates my point. Also the user can put there features in any folder they want. It doesn't have to be in a `features` folder.